src/passes.jl

```diff
@@ -306,6 +306,7 @@ end
 function process!(::SetupBuildDirectory, page::Page, content, block)
     dir = dirname(page.build)
     isdir(dir) || mkpath(dir)
+    addstaticfiles!(dir)
```

Comment:
No! Ok, I thought the false at the end of it was something to do with that.

I had this somewhere else in a first pass but suspected you would feel strongly where it should go.... It feels kinda strange to have it as a pass (for every page for once time things like this).

The other option was to append! staticfiles to document.pages, that's probably the most elegant (but I couldn't get it working).